packages/patternfly-4/react-core/src/components/Modal/ModalContent.tsx

```diff
@@ -29,6 +30,8 @@ export interface ModalContentProps {
   isOpen?: boolean;
   /** Complex header (more than just text), supersedes title for header content */
   header?: React.ReactNode;
+  /** Description */
+  description?: React.ReactNode;
```

Comment:
SHouldn't we also add this prop to `Modal` so the consumer knows that is where they pass it?